kubetest/main.go

```diff
@@ -181,6 +182,7 @@ func defineFlags() *options {
 	flag.DurationVar(&timeout, "timeout", time.Duration(0), "Terminate testing after the timeout duration (s/m/h)")
 	flag.BoolVar(&o.up, "up", false, "If true, start the e2e cluster. If cluster is already up, recreate it.")
 	flag.StringVar(&o.upgradeArgs, "upgrade_args", "", "If set, run upgrade tests before other tests")
+	flag.IntVar(&o.boskosWaitDuration, "boskos-wait-duration", 5, "Defines how long it waits until quite getting Boskos resoure, default 5 minutes")
```

Comment:
can you make it a DurationVar instead? :-)

Reply:
(also, s/quite/quit?)

Reply:
done, both. Good catch!